index.html

```diff
@@ -85,17 +85,17 @@ <h3>Deep linking</h3>
 								scrolling="no"
 								horizontalscrolling="no"
 								verticalscrolling="no"
-								src="https://www.amilia.com/store/en/usa-amilia-group/shop">
+								src="https://app.amilia.com/store/en/usa-amilia-group/shop">
 						</iframe>
-						<script src="https://www.amilia.com/scripts/amilia-iframe.js" type="text/javascript"></script>
+						<script src="https://app.amilia.com/scripts/amilia-iframe.js" type="text/javascript"></script>
 					</div>
 					<!-- End of Amilia embed -->
 				</div>
 			</div>
 		</div>
 		<footer>
 			<div class="container">
-				<p class="pull-left">&copy; 2021 <a href="https://www.amilia.com/en">Amilia Inc.</a></p>
+				<p class="pull-left">&copy; 2021 <a href="https://app.amilia.com/en">Amilia Inc.</a></p>
```

Comment:
Here it should be `www`, because we want to target the marketing website.
```suggestion
				<p class="pull-left">&copy; 2021 <a href="https://www.amilia.com/en">Amilia Inc.</a></p>
```